documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,183 @@
+* You have a GitHub account. See [Signing up for a new GitHub account](https://help.github.com/en/articles/signing-up-for-a-new-github-account).
+* You have access to a repository on GitHub. See [Creating a repository on GitHub](https://help.github.com/en/articles/creating-a-repository-on-github).
+* You have an account and space in the Cloud Foundry environment on SAP Cloud Platform. See [Get Started with a Trial Account: Workflow in the Cloud Foundry Environment](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/e50ab7b423f04a8db301d7678946626e.html).
+
+## (Optional) Install the `cx-server` Toolkit for Jenkins
+
+`cx-server`is a lifecycle management toolkit that provides Docker images with a preconfigured Jenkins and a Nexus-based cache to facilitate the configuration and usage of Jenkins.
+
+To use the toolkit, get the `cx-server` script and its configuration file `server.cfg` by using the following command:
+
+```sh
+docker run -it --rm -u $(id -u):$(id -g) -v "${PWD}":/cx-server/mount/ ppiper/cx-server-companion:latest init-cx-server
+```
+
+When the files are downloaded into the current directory, launch the Jenkins server by using the following command:
+
+
+```sh
+./cx-server start
+```
+
+For more information on the Jenkins lifecycle management and how to customize your Jenkins, have a look at the [Operations Guide for Cx Server](https://github.com/SAP/devops-docker-images/blob/master/docs/operations/cx-server-operations-guide.md). 
+
+
+## Create Your First Pipeline
+
+For the beginning, we recommend using an SAP sample application. The repository [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] contains a simple `nodejs` application that can be enriched with a pipeline built with MTA and deployed into the Cloud Foundry environment.
+
+1. Fork the [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] repository into your GitHub organization.
+
+1. Select the `1_REST_persist_in_Memory` branch of your [cloud-cf-helloworld-nodejs] fork and in it, create a new file with the name `Jenkinsfile`.
+
+1. To synchronize the repository, enter the following code lines into your `Jenkinsfile`: 
+   ```
+   @Library('piper-lib-os') _
+    node() {
+    stage('prepare') {
+      checkout scm
+      setupCommonPipelineEnvironment script:this
+      }
+    }
+   ```
+   For more information about Jenkinsfiles and pipelines, see [Using a Jenkinsfile][jenkins-io-jenkinsfile].
+   
+1. To set up a Jenkins job for your repository, open the Jenkins UI under `http://<jenkins-server-address>:<http-port>` and choose **New Item**. Per default, the `cx-server` starts Jenkins on HTTP port `80`. For more information, see the [Jenkins User Documentation][jenkins-io-documentation].
+   <p align="center">
+   ![Clicke New Item](../images/JenkinsHomeMenu-1.png "Jenkins Home Menu")
+   </p> 
+   
+1. Provide a name for your new item (for example, *My First Pipeline*) and select **Pipeline**.
```

Comment:
> My First Pipeline

I did not find an issue directly. But from exerience in the past I would like to suggest to have a name without blanks, since the names are used as file system pathes. Theoretically that should not cause issues, but ...

Reply:
Regarding the guided tour, it works. We dont know whether the customer uses blank for is pipeline but we have to make sure, that even these pipelines succeeds. dont we have test cases for this? btw: we already had this issue (with newman?) and fixed it.

Furthermore, i'd need to adjust all pictures....

Reply:
i remember that we had this issue once and we fixed it. dont we have tests for it? from my point of view, we shouldn't worry. furthermore, i'd have to recreate a couple of images...